### ISSUE TITLE
error in demo 12

### ISSUE THREAD

**Issue by choper725** [abap2UI5/abap2UI5#444]
![image](https://github.com/abap2UI5/abap2UI5/assets/129428114/8e51ba35-3299-41ae-b65b-c81ed36684b2)

when pressing

Demo in new app: popup rendering, no background

**Comment by oblomov-dev** [comment]
fixed with https://github.com/abap2UI5/abap2UI5/pull/445

**Comment by choper725** [comment]
regarding popups..
is there a way to use client->view_model_update( ).
in a popup ?

im trying to get it to work on Search button in demo09 ('POPUP_TABLE_F4_CUSTOM').

**Comment by oblomov-dev** [comment]
Yes you can use popup_model_update for this

**Comment by choper725** [comment]
oh didnt notice that a thing..

im trying to get to f4 popup demo to be more fluent, there is too much screen rendering in that demo..

**Comment by oblomov-dev** [comment]
yes it's good to ask, the demos are created over the time and sometimes they are not completely up-to-date anymore or too complicated because the framework changed and makes certain thing easier now.

you were right mv_main_view is not used, deleted it and also made the demo more clear.

Avoiding too much rerendering is also what i prefer, for every ui element:
view
popup
popover
nested_view 

you have the same three methods:
destroy
display
model_update

Makes it easier to rerender or update only the specific element you are changing at the moment.

And the event on_navigated is true is situations when:
initial load
leave to app
call app

that helps you to know when you have to rerender the whole view otherwise after app changes you will run into situations where no view is shown.

hope that helps a little bit, if you think some demos are too complicated just let me know that we can adjust them.
